ceph-nvmeof.conf

```diff
@@ -10,7 +10,7 @@
 [gateway]
 name =
 group =
-addr = 192.168.13.3
+addr = 0.0.0.0
```

Comment:
I've been trying to make this work and miserably failed when adding the listener (as the gateway uses this `0.0.0.0` as the listener IP if no other address is specified).

I see that the cli tests:
* `["-g", gateway_name, "-a", addr, "-s", "5001"]`: this won't use the `0.0.0.0` address but the specified one.
* `["-s", "5002"]`: however, I'm surprised that this will work, since it'll default to `0.0.0.0` and, at least in my case, it repeatedly failed.

But it looks like it works:
```
[2023-07-23 22:19:22.988356] tcp.c: 850:nvmf_tcp_listen: *NOTICE*: *** NVMe/TCP Target Listening on 127.0.0.1 port 5001 ***
[2023-07-23 22:19:23.000319] tcp.c: 850:nvmf_tcp_listen: *NOTICE*: *** NVMe/TCP Target Listening on 0.0.0.0 port 5002 ***
```
Have you tried `nvme connect` with both? I think I didn't manage to get it working with the `0.0.0.0` address.

Reply:
I already tried `nvme connect` inside GH Actions runner, but it failed (it actually failed at the `modprobe nvme-fabrics`). I couldn't debug that, but in order to debug GH Actions, I added this step (currently commented out) `Setup tmate session`, which allows to pause a job and run an SSH session in the runner, in order to debug/troubleshoot.